packages/elements/src/flag/index.ts

```diff
@@ -114,6 +116,19 @@ export class Flag extends BasicElement {
     }
   }
 
+  /**
+   * A deferred promise representing flag ready.
+   * It would be resolved when the flag svg has been fetched and parsed, or
+   * when the flag svg is unavailable/invalid.
+   */
+  private flagReady!:Deferred<void>;
+
+  constructor () {
+    super();
+    this.flagReady = new Deferred<void>();
+    this.flagReady.resolve();
```

Comment:
How resolve flagReady in constructor difference from not resolve?

Reply:
Clarification comment added. Hope this clears things up ;)